decentraland/unity-renderer
# feat: leave channel error modal

## What does this PR change?

Adds a new modal that is displayed whenever there is an error to leave a channel.

## How to test the changes?

1. Easiest way to test is to force `ChatController.LeaveChannel` method to trigger `OnChannelLeaveError` event
2. Open the conversation list
3. Leave any channel
4. A modal should be displayed with the option to close or retry

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
